app/controllers/api/projects.php

```diff
@@ -1237,7 +1237,7 @@
         }
 
         $platforms = $dbForConsole->find('platforms', [
-            Query::equal('projectId', [$project->getId()]),
+            Query::equal('projectInternalId', [$project->getInternalId()]),
```

Comment:
So the internal project ID is different even if a project is recreated with the same ID so this would prevent any old platform from ever being fetched, right?